app/src/componentLibrary/sidebar/sidebar.jsx

```diff
@@ -32,20 +33,30 @@ export const Sidebar = ({
   topSidebarItems,
   bottomSidebarItems,
   createFooterBlock,
+  isOpenPopover,
   ...navbarProps
 }) => {
-  const [isOpenNavbarPopover, setIsOpenNavbarPopover] = useState(false);
   const [isOpenNavbar, setIsOpenNavbar] = useState(false);
 
   const [actionButtonKey, setActionButtonKey] = useState(null);
   const [actionButtonType, setActionButtonType] = useState(null);
 
+  const sidebarRef = useRef(null);
+
+  const handleClickOutside = useCallback(() => {
+    if (isOpenNavbar) {
+      setIsOpenNavbar(false);
+    }
+  }, [isOpenNavbar]);
+
+  useOnClickOutside(sidebarRef, handleClickOutside);
+
   const onOpenNavbar = () => {
     setIsOpenNavbar(true);
   };
 
   const onCloseSidebar = () => {
```

Comment:
This handler seems to be associated with `navbar` rather than `sidebar`.
Please rename it accordingly.
![image](https://github.com/reportportal/service-ui/assets/31861805/37ff1af2-98f2-401a-be27-a7df7f573431)

Reply:
Done.